packages/frontend/src/components/create-campaign/preview/index.tsx

```diff
@@ -52,6 +52,11 @@ export function CampaignPreview({
     const { address: connectedAddress } = useAccount();
     const { writeContractAsync } = useWriteContract();
 
+    const campaignDurationDays = useMemo(() => {
+        if (!payload.startDate) return 0;
```

Comment:
One thing here would be that no one guarantees us that the campaign will last more than a day. So for that look how I did it in the campaign row in the all campaigns page. That's more flexible.

Reply:
Fixed in a06f031f998ba0c7d995b892b4e399b596601e9d.